Week3/todoappReadme.md

```diff
@@ -0,0 +1,110 @@
+# ToDoApp Readme (Abdullah)
```

Comment:
thanks for the read me 👍 
It helped me a lot to understand how to use the code

Reply:
👍